README.adoc

```diff
@@ -182,9 +189,12 @@ When the service is ready, you will see an output similar to the one below. The
 }
 ----
 
-The JSON output has a `highest` attribute which represents the system with the highest load. Similarly, the `lowest` attribute represents the system with the lowest load. The JSON output for each of these feature the respective hostname and systemLoad of the system.
+The JSON output contains a `highest` attribute that represents the system with the highest load.
+Similarly, the `lowest` attribute represents the system with the lowest load. 
+The JSON output for each of these attributes features the respective `hostname` and `systemLoad` of the system.
 
-When you are done checking out the application, run the following command to stop the query microservice. We will leave the `system` and `inventory` services running because we will use them when we rebuild the application later on. 
+When you are done checking out the application, run the following command to stop the `query` microservice. 
+We will leave the `system` and `inventory` services running because they will be used when the application is rebuilt later in the guide:
```

Comment:
>We will leave the `system` and `inventory` services running because they will be used when the application is rebuilt later in the guide:

To

>Leave the `system` and `inventory` services running because they will be used when the application is rebuilt later in the guide: